tests/models/test_dagbag.py

```diff
@@ -151,6 +151,42 @@ def test_zip(self):
         dagbag.process_file(os.path.join(TEST_DAGS_FOLDER, "test_zip.zip"))
         self.assertTrue(dagbag.get_dag("test_zip_dag"))
 
+    @conf_vars({('core', 'max_tasks_per_dag'): '5'})
+    def test_process_file_max_task_check(self):
+        """
+        test if num_tasks > max_tasks_per_dag can be identified
+        """
+        a_dag_id = "example_short_circuit_operator"
```

Comment:
```suggestion
        a_dag_id = "test_example_bash_operator"
```

I take a quick look. `TEST_DAGS_FOLDER` point to `airflow/tests/models/../dags` and this path not including dag `example_short_circuit_operator`. So you could not process_file `airflow/tests/models/../dags/example_short_circuit_operator.py`

Maybe you should use others dag in `airflow/tests/models/../dags`(which `TEST_DAGS_FOLDER` point to), just like `test_example_bash_operator` and change `@conf_vars({('core', 'max_tasks_per_dag'): '5'})` to `@conf_vars({('core', 'max_tasks_per_dag'): '7'})`.

So as others test